boards/st/stm32h750b_dk/stm32h750b_dk.dts

```diff
@@ -168,8 +175,37 @@
 	};
 };
 
-&rtc {
-	clocks = <&rcc STM32_CLOCK_BUS_APB4 0x00010000>,
-		 <&rcc STM32_SRC_LSE RTC_SEL(1)>;
+&ltdc {
+	pinctrl-0 = <&ltdc_r0_pi15 &ltdc_r1_pj0 &ltdc_r2_pj1 &ltdc_r3_ph9
+		&ltdc_r4_pj3 &ltdc_r5_pj4 &ltdc_r6_pj5 &ltdc_r7_pj6
+		&ltdc_g0_pj7 &ltdc_g1_pj8 &ltdc_g2_pj9 &ltdc_g3_pj10
+		&ltdc_g4_pj11 &ltdc_g5_pi0 &ltdc_g6_pi1 &ltdc_g7_pk2
+		&ltdc_b0_pj12 &ltdc_b1_pj13 &ltdc_b2_pj14 &ltdc_b3_pj15
+		&ltdc_b4_pk3 &ltdc_b5_pk4 &ltdc_b6_pk5 &ltdc_b7_pk6
+		&ltdc_de_pk7 &ltdc_clk_pi14 &ltdc_hsync_pi12 &ltdc_vsync_pi9>;
+	pinctrl-names = "default";
+	/* disp-on-gpios = <&gpioi 12 GPIO_ACTIVE_HIGH>; */
+	/* bl-ctrl-gpios = <&gpiok 3 GPIO_ACTIVE_HIGH>; */
```

Comment:
Question on these commented lies:
Will they be potentially useful ? 
   If yes, under which conditions ? -> A comment would be nice
  if not, can we remove them ?